finance/server/src/main/java/com/google/finapp/ServerMain.java

```diff
@@ -75,5 +81,10 @@ private static class Args {
         names = {"--spanner_use_jdbc"},
         arity = 0)
     boolean spannerUseJdbc = false;
+
+    @Parameter(
+        names = {"--spanner_use_pg"},
+        arity = 0)
+    boolean spannerUsePg = false;
```

Comment:
Would it make sense to merge the use_pg with use_jdbc and make it an enum?

e.g: spanner_interface one_of{gsql_java, gsql_jdbc, pg_java}

Among the issues now is that you can actually use the --spanner_use_jdbc=true with --spanner_use_pg=true and nothing complains.